tests/acceptance/00_basics/def.json/concurrent-vars-and-variables-keys/def.json

```diff
@@ -0,0 +1,10 @@
+{
+    "vars": {
+        "my_var": "Set by vars key",
+        "in_vars": "Only in vars"
+    },
+    "variables": {
```

Comment:
Does the test still pass if variables is placed before vars in def.json? Just wondering how the file is processed, if it is "in order" and that's why the variables overrides vars.

Reply:
It does (I tested it manually), but I can add a version explicitly for that.

Reply:
Added commit with another version of the test. Waiting for travis.